qiskit/circuit/library/__init__.py

```diff
@@ -74,52 +158,28 @@
      [0.+0.j 0.+0.j 1.+0.j 0.+0.j]
      [0.+0.j 1.+0.j 0.+0.j 0.+0.j]]
 
+
+The function :func:`.get_standard_gate_name_mapping` allows you to see the available standard gates
+and operations.
+
+.. autofunction:: get_standard_gate_name_mapping
+
+1-qubit standard gates
+----------------------
+
 .. autosummary::
    :toctree: ../stubs/
-   :template: autosummary/class_no_inherited_members.rst
 
-   C3XGate
-   C3SXGate
-   C4XGate
-   CCXGate
-   DCXGate
-   CHGate
-   CPhaseGate
-   CRXGate
-   CRYGate
-   CRZGate
-   CSGate
-   CSdgGate
-   CSwapGate
-   CSXGate
-   CUGate
-   CU1Gate
-   CU3Gate
-   CXGate
-   CYGate
-   CZGate
-   CCZGate
-   ECRGate
    HGate
    IGate
    MSGate
```

Comment:
MSGate is not a 1-qubit gate? Is it even a standard gate?

Reply:
it's true that `MSGate` is no longer returned from `get_standard_gate_name_mapping`, maybe it was considered standard at some point and that's why it's still in the docs. I believe this should be moved to the `generalized gates` section.